lib/credit_card_sanitizer.rb

```diff
@@ -23,8 +23,8 @@ class CreditCardSanitizer
   }
   VALID_COMPANY_PREFIXES = Regexp.union(*CARD_COMPANIES.values)
   EXPIRATION_DATE = /\s(?:0?[1-9]|1[0-2])(?:\/|-)(?:\d{4}|\d{2})(?:\s|$)/
-  LINE_NOISE = /[^\w_\n,()\/:]{,5}/
-  SCHEME_OR_PLUS = /((?:&#43;|\+)|(?:[a-zA-Z][\-+.a-zA-Z\d]{,9}):\S+)/
+  LINE_NOISE = /[^\w_\n,()\/:;]{,5}/
+  SCHEME_OR_PLUS = /(\+|(?:[a-zA-Z][\-+.a-zA-Z\d]{,9}):\S+)/
```

Comment:
This change was still good, since a `+` in HTML entity format does indicate a phone number, and we might still redact what comes afterward as a false positive